interactions/api/models/misc.py

```diff
@@ -1,8 +1,10 @@
 # TODO: This is post-v4.
-# TODO: Reorganise these models based on which big obj uses little obj
-# TODO: Potentially rename some model references to enums, if applicable
-# TODO: Reorganise mixins to its own thing, currently placed here because circular import sucks.
+#   Reorganise these models based on which big obj uses little obj
+#   Potentially rename some model references to enums, if applicable
```

Comment:
Have these TODOs actually been completed? Removing the convention doesn't suddenly make it no longer a task.

Reply:
No, but you only need one TODO-statement. If you indent the other lines, they will be counted as a whole TODO-statement

Reply:
![grafik](https://user-images.githubusercontent.com/88881326/161830558-a9540306-a190-400f-ba11-a9d8dff298d0.png)